lib/std/os/windows.zig

```diff
@@ -602,6 +602,8 @@ pub const ReadFileError = error{
     OperationAborted,
     /// Unable to read file due to lock.
     LockViolation,
+    /// Unable to read from disconnected virtual com port
+    AccessDenied,
```

Comment:
This feels like too specific of a description. Maybe something like this instead?

```suggestion
    /// Known to be possible when:
    /// - Unable to read from disconnected virtual com port (Windows)
    AccessDenied,
```

Alternatively, it might make sense to avoid the doc comment in the error set and instead put this sort of comment at the error handling site:

```zig
    // Known to be possible when trying to read from a disconnected virtual com port
    .ACCESS_DENIED => return error.AccessDenied,
```

(and same idea for `WriteFile`)

Reply:
Makes sense, I've kept the comments in the error sets just because existing comments are there rather than in the switch statements.